src/Our.Umbraco.AuthU/Data/Plans/RegisterOAuthTables.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Our.Umbraco.AuthU.Data.Helpers;
+using Our.Umbraco.AuthU.Data.Migrations.UmbracoDbOAuthClientStore;
+using Our.Umbraco.AuthU.Data.Migrations.UmbracoDbOAuthRefreshTokenStore;
+using Our.Umbraco.AuthU.Models;
+using System;
+using System.Linq;
+using Umbraco.Core.Composing;
+using Umbraco.Core.Migrations;
+
+namespace Our.Umbraco.AuthU.Data.Migrations.Plans
+{
+    /// <summary>
+    /// This is the plan that runs the AuthU Migrations
+    /// </summary>
+    internal class RegisterOAuthTables : MigrationPlan
+    {
+        /// <summary>
+        /// Add the demo id if preferred
+        /// </summary>
+        /// <param name="CreateDemoClient"></param>
+        public RegisterOAuthTables(bool CreateDemoClient = false) : base("AuthU")
```

Comment:
`CreateDemoClient` is never true. I think we could probably just remove this param and just always install demo client. If people don't use it, it's not a big deal.